docs/Migration-from-the-AuthComponent.md

```diff
@@ -151,3 +151,36 @@ $this->Authentication->allowUnauthenticated(['view']);
 ```
 
 Each call to `allowUnauthenticated()` will overwrite the current action list.
+
+# Migrating Unauthorized Redirects
+
+By default `AuthComponent` redirects users back to the login page when
+authentication is required. In contrast, the `AuthenticationComponent` in this 
+plugin will raise an exception in this scenario. You can convert this exception
+into a redirect by using the `UnauthorizedRedirectMiddleware` in your application.
+
+```php
+// in src/Application.php
+use Authentication\Middleware\AuthenticationMiddleware;
+use Authentication\Middleware\UnauthorizedRedirectMiddleware;
+
+public function middleware($middlewareQueue)
+{
+    // Various other middlewares for error handling, routing etc. added here.
+
+    // Add the authentication middleware
+    $authentication = new AuthenticationMiddleware($this);
+
+    // Add authentication
+    $middlewareQueue->add($authentication);
+
+    // Add unauthorized redirect handling
+    $middlewareQueue->add(new UnauthorizedRedirectMiddleware('/users/login'));
```

Comment:
Can we reuse a central config here inside the middleware instead? Would be more DRY, I would also prefer the default array syntax for the CakePHP use case.

Reply:
The authentication plugin doesn't have/use centralized configuration thus far.